numberformat/proposed.emu

```diff
@@ -994,8 +1050,10 @@
             1. Let _str_ be *"-0"*.
           1. Else,
             1. Let _str_ be ! Number::toString(_x_).
-        1. If the grammar cannot interpret _str_ as an expansion of |StringNumericLiteral|, return *NaN*.
-        1. Let _mv_ be the MV, a mathematical value, of ? ToNumber(_str_), as described in <emu-xref href="#sec-runtime-semantics-mv-s"></emu-xref>.
+        1. Let _text_ be ! StringToCodePoints(_str_).
+        1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
+        1. If _literal_ is a List of errors, let _mv_ be *NaN*.
+        1. Else, let _mv_ be the StringIntlMV of _literal_.
         1. If _mv_ is 0 and the first non white space code point in _str_ is `-`, return *-0*.
         1. If _mv_ is 10<sup>10000</sup> and _str_ contains `Infinity`, return *+&infin;*.
         1. If _mv_ is -10<sup>10000</sup> and _str_ contains `Infinity`, return *-&infin;*.
```

Comment:
Should this be included directly on the new StringIntlMV ? @sffc @bakkot @ryzokuken

Reply:
The `10<sup>10000</sup>` stuff is obsolete and should be removed.  It existed from a previous version of the 262 grammar that used that number to represent infinity.

Reply:
Got it it's related with https://github.com/tc39/proposal-intl-numberformat-v3/issues/69